packages/ckeditor5-package-generator/lib/index.js

```diff
@@ -31,6 +32,16 @@ const TEMPLATES_TO_FILL = [
 	'README.md'
 ];
 
+// Npm does not publish the `.gitignore` file even if it's somewhere inside the package.
+// Hence, the package generator will create it manually. See: #50.
+const GITIGNORE_ENTRIES = [
+	'build/',
+	'coverage/',
+	'node_modules/',
+	'tmp/',
+	'sample/ckeditor.dist.js'
+];
```

Comment:
```suggestion
	'tmp/',
	'sample/ckeditor.dist.js',
	''
];
```

I think there should be an empty line at the end of the file.